Bottom: 96cc1f97e4aee2e4c2cc9e1822033fd81b8ebca3
Top:    20bb643d3772961e3baded397146b545c09c3535
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-03 12:59:33 +0800

Refresh of vhost-packed-ring-support

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index f85a3f3..477494f 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1412,7 +1412,8 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 		break;
 	case VHOST_SET_VRING_USED_BASE:
 		/* Moving base with an active backend?
-		 * You don't want to do that. */
+		 * You don't want to do that.
+		 */
 		if (vq->private_data) {
 			r = -EBUSY;
 			break;
@@ -2009,8 +2010,7 @@ static int get_indirect_packed(struct vhost_virtqueue *vq,
 
 	/* Sanity check */
 	if (unlikely(len % sizeof(desc))) {
-		vq_err(vq, "Invalid length in indirect descriptor: "
-		       "len 0x%llx not multiple of 0x%zx\n",
+		vq_err(vq, "Invalid length in indirect descriptor: len 0x%llx not multiple of 0x%zx\n",
 		       (unsigned long long)len,
 		       sizeof desc);
 		return -EINVAL;
